src/main/kotlin/io/libp2p/core/protocol/Negotiator.kt

```diff
@@ -37,20 +38,34 @@ object Negotiator {
     private const val TIMEOUT_MILLIS: Long = 10_000
     private const val MULTISTREAM_PROTO = "/multistream/1.0.0"
 
-    private val MULTISTREAM_PROTO_BYTES = MULTISTREAM_PROTO.toByteArray().toByteBuf()
-    private val NEWLINE = "\n".toByteArray().toByteBuf()
-    private val NA = "na".toByteArray().toByteBuf()
+    private val HEADING = '\u0013'
+    private val NEWLINE = "\n"
+    private val DELIMITER = "\r"
+    private val NA = "na"
+    private val DELIMITERS = arrayOf(
+            wrappedBuffer("\n\r".toByteArray()),
```

Comment:
Did you find these two variants when testing against other implementations?

Reply:
Yep, that's what I was receiving from Go-echo. Do you have specs on this?